src/containers/CartPage.less

```diff
@@ -200,6 +200,69 @@
             border-color: @mainColor;
           }
         }
+
+        .btn-cmd-checkout {
+          display: inline-block;
+          padding: 8px 14px;
+          margin-bottom: 0;
+          font-size: 14px;
+          text-align: center;
+          vertical-align: middle;
+          cursor: pointer;
+          border-radius: 4px;
+          white-space: nowrap;
+          user-select: none;
+          border-radius: 2px;
+          margin-bottom: 2px;
+          color: #ffffff;
+          background-color: @mainColor;
+
+          &:focus {
+            outline: thin dotted #333;
+            outline: 5px auto -webkit-focus-ring-color;
+            outline-offset: -2px;
+          }
+
+          &:hover,
+          &:focus {
+            color: #333333;
+            text-decoration: none;
+          }
+
+          &:active,
+          &.active {
+            outline: 0;
+            background-image: none;
+            -webkit-box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+            box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+          }
+
+          &.disabled,
+          &[disabled],
+          fieldset[disabled] & {
+            cursor: not-allowed;
+            pointer-events: none;
+            opacity: 0.65;
+            filter: alpha(opacity=65);
```

Comment:
You can remove this line. Regular `opacity` (previous line) is now supported in all browsers.